app/src/main/java/com/example/therundown/domain/NbaManager.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.therundown.domain
+
+import com.example.therundown.data.Data
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object NbaManager {
```

Comment:
maybe change name to NBAClient or NBAApi

Reply:
fixed